src/test/ui/rfc-2091-track-caller/caller-location-intrinsic.rs

```diff
@@ -1,18 +1,18 @@
 // run-pass
```

Comment:
I wonder if you should add a `-Z mir-opt-level=0` revision to this (so the test is attempted both with and without MIR optimizations) - I'm worried constprop makes testing this hard.

OTOH, I think you can force this to be runtime by having *another* wrapper function around `Location::caller`. It should just work, and won't be constprop'd if it's not `const fn` (you can also put `#[inline(never)]` on it to be sure).

Reply:
Nevermind, `track-caller-attribute.rs` seems to test the runtime situation properly.